handycon.py

```diff
@@ -613,12 +619,7 @@ async def capture_keyboard_events():
                             elif active == [] and seed_event.code in [10] and button_on == 0 and button2 in event_queue:
                                 this_button = button2
                                 await do_rumble(0, 150, 1000, 0)
-
```

Comment:
Why was this removed? There is currently a bug in Steam that broke the OSK shortcut, but I'd prefer to operate under the assumption that it will be fixed.

Reply:
I removed it since the WM2 has a physical keyboard and [11] was being used to toggle the gyro.

Reply:
Oh, right. that makes sense. We can map that to another event instead. Screenshot might be a good one, or we could simplify the logic for gyro to that event only.